src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyFluentWait.java

```diff
@@ -1,5 +1,6 @@
 package com.wiley.autotest.selenium.elements.upgrade;
 
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
```

Comment:
Incorrect import, use NoSuchElementException

Reply:
it was unused. removed